Examples.xcodeproj/project.pbxproj.orig

```diff
@@ -0,0 +1,1196 @@
+// !$*UTF8*$!
```

Comment:
This looks like the product of a resolved merge conflict — should be safe to delete this file.